docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1327,23 +1327,65 @@ To enable export to parquet, an administrator must:
-|`operation,bundle`              |    unknown    |
-
-Each FHIR create, update, delete, bundle or custom operation triggers 2 CADF events - begins with an event with "pending" outcome and ends with an event with "success" or "failure" outcome; All the other FHIR operations only trigger 1 CADF event with either "success" or "failure" outcome.
-
-### 4.11.1 Enable CADF audit logging service
-Please refer to the properties names started wtih fhirServer/audit/ in [5.1 Configuration properties reference](#51-configuration-properties-reference) for how to enable and configure CADF audit logging service.
-
-### 4.11.2 Event Streams configuation of CADF audit logging service
-The CADF audit logging service gets event streams service credential from env variable EVENT_STREAMS_AUDIT_BINDING with values like this:
+The CADF audit logging service pushes FHIR server audit events for FHIR operations in [Cloud Auditing Data Federation (CADF)](https://www.dmtf.org/standards/cadf) standard format to the IBM Cloud Event Streams service. Each FHIR operation triggers a CADF audit log entry to be logged. The mapping of FHIR operation to CADF action is as follows:
+
+| FHIR Operation                                      | CADF Action |
+|-----------------------------------------------------|-------------|
+|`history,metadata,read,search,validate,version read` |   read      |
+|`create`                                             |   create    |
+|`update`                                             |   update    |
+|`delete`                                             |   delete    |
+|`bundle,custom operation,patch`                      |   unknown   |
+
+### 4.11.1 CADF audit log entry
+The following table describes the contents of a CADF audit log entry as logged by the FHIR server:
+
+| CADF Audit Log Entry Field                           | Description |
+|------------------------------------------------------|-------------|
+|`action`                                              |Action that created the audit event. Possible values are "read", "update", "create", "delete", and "unknown".|
+|`eventTime`                                           |Audit event creation timestamp.|
+|`eventType`                                           |Audit event type. Value is always "activity".|
+|`id`                                                  |Globally unique identifier for the audit event.|
+|`outcome`                                             |Action outcome. Possible values are "success", "failure", "unknown", and "pending".|
+|`typeURI`                                             |TypeURI property of the CADF event entity. Value is always "http://schemas.dmtf.org/cloud/audit/1.0/event".|
+|`attachments`                                         |Note: Contains FHIR server-specific audit event data.|
+|`attachments:contentType`                             |FHIR server-specific audit event data content type. Value is always "application/json".|
```

Comment:
I'm assuming `:` here just means its about the contentType field within attachments?
I might have expected `/` or `.` but I guess its fine?

Reply:
Yes, it's a sub-field delimiter.  I just used : since that's what CADF uses in their documentation:  https://www.dmtf.org/sites/default/files/standards/documents/DSP2038_1.1.0.pdf
I can certainly change it to something else if you prefer.

Reply:
Replaced : with /